src/Api/Management/Entity/Developer.php

```diff
@@ -91,58 +98,79 @@ public function setCompanies(string ...$companies): void
     }
 
     /**
-     * @inheritdoc
+     * {@inheritdoc}
      */
     public function hasCompany(string $companyName): bool
     {
         return in_array($companyName, $this->companies);
     }
 
     /**
-     * @inheritdoc
+     * {@inheritdoc}
      */
     public function getUserName(): ?string
     {
         return $this->userName;
     }
 
     /**
-     * @inheritdoc
+     * {@inheritdoc}
      */
     public function setUserName(string $userName): void
     {
         $this->userName = $userName;
     }
 
     /**
-     * @inheritdoc
+     * {@inheritdoc}
      */
     public function getFirstName(): ?string
     {
         return $this->firstName;
     }
 
     /**
-     * @inheritdoc
+     * {@inheritdoc}
      */
     public function setFirstName(string $firstName): void
     {
         $this->firstName = $firstName;
     }
 
     /**
-     * @inheritdoc
+     * {@inheritdoc}
      */
     public function getLastName(): ?string
     {
         return $this->lastName;
     }
 
     /**
-     * @inheritdoc
+     * {@inheritdoc}
      */
     public function setLastName(string $lastName): void
     {
         $this->lastName = $lastName;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setEmail(string $email): void
+    {
+        $this->email = $email;
+        if (null === $this->originalEmail) {
+            $this->originalEmail = $email;
+        }
+    }
+
+    /**
+     * The original email address of the developer used when updating user email address.
+     *
+     * @see Apigee\Edge\Api\Management\Controller\DeveloperController
+     */
+    public function originalEmail(): ?string
```

Comment:
Is there a reason for not having "get" as prefix?

Reply:
Yes, implementing it as `GET` causes "`Invalid JSON payload received . Unknown name \"originalEmail\" at 'developer': Cannot find field.`" error and update fails.

Reply:
Okay, so this happens because the default serialization solution in the library assumes that every public getter has to be serialized and added to the request payload.
A proper solution would be convincing the serializer not to serialize the `getOriginalEmail()`. IIRC there are examples for entity-type specific serializers in the library.

The other way around is keeping this method, marking them as `@internal` (even if it is public) and explaining in a short comment why it is not "get...".